elements/lisk-chain/src/block_header.ts

```diff
@@ -0,0 +1,286 @@
+/*
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import { objects } from '@liskhq/lisk-utils';
+import { signDataWithPrivateKey, hash } from '@liskhq/lisk-cryptography';
+import { codec } from '@liskhq/lisk-codec';
+import { validator, LiskValidationError } from '@liskhq/lisk-validator';
+import { TAG_BLOCK_HEADER } from './constants';
+
+export const signingBlockHeaderSchema = {
```

Comment:
should we move these schemas to `schema.ts`?

Reply:
Yes we should, when we integrate new block header. If done now, will break a lot of things in the framework and other places.